lib/siwapp/templates.ex

```diff
@@ -276,6 +276,24 @@ defmodule Siwapp.Templates do
     EEx.eval_string(template, eval_data)
   end
 
+  @spec sanitize_invoice(Siwapp.Invoices.Invoice.t()) :: Siwapp.Invoices.Invoice.t()
+  defp sanitize_invoice(invoice) do
+    fields_to_sanitize = [
+      :name,
+      :identification,
+      :email,
+      :contact_person,
+      :invoicing_address,
+      :shipping_address,
+      :notes,
+      :terms
+    ]
+
+    Enum.reduce(fields_to_sanitize, invoice, fn field, acc ->
+      %{acc | field => HtmlSanitizeEx.strip_tags(Map.get(acc, field))}
+    end)
```

Comment:
please, do not use reduce when you are not reducing anything. 
You can see the original PR implementation to change for a more proper option.

Reply:
I think that is not a bad use for `Enum.reduce`, but if you think that I will change it